tool2schema/schema.py

```diff
@@ -183,12 +185,19 @@ def _get_parameters_schema(self, schema_type: SchemaType) -> dict:
         """
         schema = {"type": "object"}
 
-        if self.parameter_schemas or schema_type == SchemaType.TUNE:
-            # If the schema type is tune, add the dictionary even if empty
-            schema["properties"] = self._get_parameter_properties_schema()
+        if (
+            not self.parameter_schemas
+            and schema_type == SchemaType.API
+            or self.config.ignore_all_parameters
+        ):
+            # Skip properties
+            return schema
+
+        # If the schema type is tune, add the dictionary even if empty
+        schema["properties"] = self._get_parameter_properties_schema()
```

Comment:
If `self.config.ignore_all_parameters` is true, then we do not populate `schema["properties"]` with {} even if the `schema_type` is `TUNE`?

Reply:
Yes, should be fixed with 5570e21, now `parameter_schemas` returns an empty dictionary if `ignore_all_parameters` is true